pmd-java/src/main/resources/category/java/codestyle.xml

```diff
@@ -1461,6 +1461,8 @@ Fields, local variables, enum constant names or parameter names that are very sh
  [not(parent::CatchParameter)]
  (: Lambda expression parameter :)
  [not(parent::LambdaParameter)]
+ (: Exclude Unnamed Variables (JEP 456) :)
+ [not(@Name='_')]
```

Comment:
probably better to use `[note(@Unnamed)]`

Reply:
Fixed in b0ac603501601f1833074652fcd6cdb0a65ec792. Note, I had to use `[@Unnamed = false()]`. For some reason, `not()` doesn't work with our boolean values...